From 760bbd33257aab232712c602c53dbb54bc74f919 Mon Sep 17 00:00:00 2001
From: Neal Liu <neal_liu@aspeedtech.com>
Date: Wed, 10 Sep 2025 11:17:49 +0800
Subject: [PATCH 05/10] arch: arm: dts: ast2700-evb: add mailbox bootmcu device
 node

Add mailbox bootmcu device node.

Signed-off-by: Neal Liu <neal_liu@aspeedtech.com>
Change-Id: I5ff1eff49692a34eb65b4fe45b129487caaca984
(cherry picked from commit 5e36f11fa13583d695d68f4c6c9692791be821d5)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 arch/arm/dts/ast2700-evb.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/dts/ast2700-evb.dts b/arch/arm/dts/ast2700-evb.dts
index 4596e307c8..524536dc2b 100644
--- a/arch/arm/dts/ast2700-evb.dts
+++ b/arch/arm/dts/ast2700-evb.dts
@@ -304,3 +304,11 @@
 	};
 };
 #endif
+
+&soc0 {
+	mbox-bootmcu-1 {
+		compatible = "aspeed,aspeed-mbox";
+		reg = <0x4 0x31880000 0x0 0x100000>, <0x4 0x31980000 0x0 0x100000>;
+		mboxes = <&mbox2 1>;
+	};
+};
-- 
2.34.1

